packages/cloudflare/src/utils/generate-routes-json.ts

```diff
@@ -0,0 +1,267 @@
+import type { AstroConfig, RouteData, RoutePart } from 'astro';
+
+import { existsSync } from 'node:fs';
+import { writeFile } from 'node:fs/promises';
+import { posix } from 'node:path';
+import { fileURLToPath } from 'node:url';
+import glob from 'tiny-glob';
+import { removeLeadingForwardSlash, removeTrailingForwardSlash } from './assets.js';
+
+function prependForwardSlash(path: string) {
+	return path[0] === '/' ? path : `/${path}`;
+}
```

Comment:
You should use `@astrojs/internal-helpers`, so you use the same code that `astro` uses to handle paths